src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTest.java

```diff
@@ -264,7 +264,11 @@ private static Stream<Arguments> validCodecToCQLTestCasesDatetime() {
         Arguments.of(
             DataTypes.TIMESTAMP,
             TEST_DATA.TIMESTAMP_VALID_STR,
-            Instant.parse(TEST_DATA.TIMESTAMP_VALID_STR)));
+            Instant.parse(TEST_DATA.TIMESTAMP_VALID_STR)),
```

Comment:
Only "to CQL" test (from JSON into CQL value for driver) because reverse (from CQL to JSON) will always use "plain" notation, not EJSON wrapper.